config/settings.yml

```diff
@@ -1014,7 +1014,6 @@
   :prefetch_max_per_worker_dequeue: 100
   :prefetch_min_per_worker_dequeue: 10
   :prefetch_stale_threshold: 30.seconds
-  :rails_server: puma
```

Comment:
Do we need a data migration to drop the setting if it was changed previously?

Reply:
we will have not used record without migration, so: https://github.com/ManageIQ/manageiq-schema/pull/416